x-pack/plugins/fleet/common/types/models/epm.ts

```diff
@@ -531,8 +531,8 @@ export type PackageList = PackageListItem[];
 export type PackageListItem = Installable<RegistrySearchResult> & {
   id: string;
   integration?: string;
-  installationInfo?: InstallationInfo;
   savedObject?: InstallableSavedObject;
```

Comment:
is `savedObject` still needed in `PackageListItem` type?

Reply:
Yes it still needed that interface is used to type the package server side, and we only populate `installationInfo` in the handler for public usage.